lib/server.js

```diff
@@ -13,7 +13,12 @@ const server = new Hapi.Server({
 server.connection({ port: Config.PORT });
 
 server.register([
-  require('hapi-bookshelf-serializer')
+  require('hapi-bookshelf-serializer'),
+  {
+    register: require('hapi-format-error'),
+    options: { joiStatusCode: 422 }
+  },
+  require('../lib/plugins/features/movies')
```

Comment:
Looks like you have some unnecessary path traversals. Since we're in `./lib`, you just have to use `./plugins/features/movies`.